evap/evaluation/migrations/0148_evaluation_main_language.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 5.1.3 on 2024-11-25 20:19
+
+from django.db import migrations, models
+
+
+def _migrate(apps, _schema_editor):
+    _approved_status = 40
+    Evaluation = apps.get_model("evaluation", "Evaluation")
+    Evaluation.objects.filter(state__gte=_approved_status).update(main_language="en")
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("evaluation", "0147_unusable_password_default"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="evaluation",
+            name="main_language",
+            field=models.CharField(
+                choices=[("en", "English"), ("de", "Deutsch"), ("x", "undecided")],
+                default="x",
+                max_length=2,
+                verbose_name="main language",
+            ),
+        ),
+        migrations.RunPython(_migrate),
```

Comment:
To make this migration reversible:
```suggestion
        migrations.RunPython(_migrate, reverse_code=migrations.RunPython.noop),
```